index.html

```diff
@@ -4101,12 +4101,16 @@ <h2>Keywords and text strings</h2>
           <p>For the Creation Time keyword, the date format defined in section&#160;5.2.14 of RFC 1123 is suggested, but not
           required [[rfc1123]].</p>
 
-          <p>In the <a class="chunk" href="#11tEXt">tEXt</a> and <a class="chunk" href="#11zTXt">zTXt</a> chunks, the text string
-          associated with a keyword is restricted to the Latin-1 character set plus the linefeed character. Text strings in
-          <a class="chunk" href="#11zTXt">zTXt</a> are compressed into <a>zlib</a> datastreams using <a>deflate</a> compression
-          (see <a href='#10CompressionOtherUses'></a>). The <a class="chunk" href="#11iTXt">iTXt</a> chunk can be used to convey
-          characters outside the Latin-1 set. It uses the UTF-8 encoding [[rfc3629]]. There is an option to compress text strings
-          in the <a class="chunk" href="#11iTXt">iTXt</a> chunk.</p>
+          <p>The <a class="chunk" href="#11iTXt">iTXt</a> chunk uses the UTF-8 encoding [[rfc3629]] and 
```

Comment:
Wasn't this [already dealt with?](https://github.com/w3c/PNG-spec/pull/252/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R3935-R3944)

Reply:
Yes. I pulled your changes. There were conflicts so I needed to pull and resolve them.
Strangely, Git / GitHub is showing it as part of this list of commits?

Reply:
If you look at the [Files changed](https://github.com/w3c/PNG-spec/pull/248/files) tab, it is only showing changes made for this pull request.